.orchestra/support/revng-container/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM docker.io/library/ubuntu:22.04 as BASE
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+COPY apt.no-extra /etc/apt/apt.conf.d/no-extra
+RUN apt update && \
+    apt install -y \
+        # Needed for recompilation pipe (-dev included due to linking problems)
```

Comment:
Let's be more accurate: we need libc6-dev for `crt*.o` files (which ones?) which are provided by the libc, which we do not install (as anything in `link-only`).

Reply:
Added comments for each dependency installed so that it's know why they are there